packages/app-staking/src/index.tsx

```diff
@@ -86,16 +86,29 @@ class App extends React.PureComponent<Props, State> {
         </header>
         <Component
           balances={validatingBalances}
+          balanceArray={this.balanceArray}
           intentions={intentions}
           validators={validators}
         />
       </main>
     );
   }
 
-  onMenuChange = (action: Actions) => {
+  private onMenuChange = (action: Actions) => {
     this.setState({ action });
   }
+
+  private balanceArray (_address: AccountId | string, _balances: RxBalanceMap): Array<Balance> | undefined {
```

Comment:
Not 100% sure we need to pass the balances through as well - we have the balances here as part of `validatingBalances`